roles/keystone-setup/tasks/domains.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: keystone create multi domains
+  command:  >
+   openstack
+   --os-identity-api-version 3
+   --os-auth-url "{{ endpoints.keystone.url.admin }}/v3"
+   --os-username admin
+   --os-password "{{ secrets.admin_password }}"
+   --os-user-domain-name default
+   --os-project-domain-name default
+   --os-project-name admin
+   domain create "{{ item }}"
+  with_items: "{{ keystone.multi_domain.domains }}"
```

Comment:
Once we get the Ansible 2.0 PR merged, we should make use of https://github.com/ansible/ansible-modules-extras/blob/devel/cloud/openstack/os_keystone_domain.py here.